release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -74,6 +74,12 @@ In {product-title} {product-version}, the installation program uses Cluster API
 
 Additionally, SSH access to control plane and compute machines is no longer open to the machine network, but is restricted to the security groups associated with the control plane and compute plane machines.
 
+[id="ocp-4-16-installation-and-update-gcp-capi_{context}"]
+==== Cluster API replaces Terraform for {gcp-first} installations (Technology Preview)
+In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {gcp-short}. This feature is available as a Technology Preview in {product-title} {product-version}. To enable this feature, set the `featureSet: TechPreviewNoUpgrade` parameter in the `install-config.yaml` file before installation.
```

Comment:
Alternatively, insert below section in install-config.yaml, which will enable ClusterAPIInstall only, rather than all tech preview features. WDYT? Thanks! 

```
featureSet: CustomNoUpgrade
featureGates:
- ClusterAPIInstall=true

Reply:
Good idea, thank you